ghost/core/core/server/models/invite.js

```
@@ -62,6 +63,7 @@ Invite = ghostBookshelf.Model.extend({
         }
 
         // CASE: make sure user is allowed to add a user with this role
+        console.log(`labs.isSet('superEditors')`, labs.isSet('superEditors'));
```

ESLint

[error] Unexpected console statement.

(no-console)